openwisp_monitoring/device/api/serializers.py

```diff
@@ -20,6 +26,77 @@ class Meta:
         model = DeviceMonitoring
 
 
+class WifiClientSerializer(FilterSerializerByOrgManaged, ValidatedModelSerializer):
+    class Meta:
+        fields = [
+            "mac_address",
+            "vendor",
+            "ht",
+            "vht",
+            "wmm",
+            "wds",
+            "wps",
+            "modified",
+            "created",
+        ]
+        model = WifiClient
+        read_only_fields = (
+            'created',
+            'modified',
+        )
+
+
+class WifiSessionCreateUpdateSerializer(
+    FilterSerializerByOrgManaged, ValidatedModelSerializer
+):
```

Comment:
add here:

```python
organization_lookup = 'wifisession__device__organization__in'
```

Please add a test for this error (which shall fail without the fix suggested above), you need to do the request to the API specifying the `HTTP_ACCEPT='text/html'`:

```python
response = self.client.get(url, HTTP_ACCEPT='text/html')
```